.storybook/config.js

```diff
@@ -12,8 +13,10 @@ function loadStories() {
 addDecorator(story => (
   /* eslint-disable react/jsx-filename-extension */
   <Fragment>
-    <GlobalStyle />
-    {story()}
+    <ServiceContextProvider service="news">
```

Comment:
Will this prevent us from seeing the right typography for Persian captions too?

Reply:
I believe it shouldn't, But lemmy do a quick test on it.